src/annotate.spec.ts

```diff
@@ -133,6 +132,7 @@ test('Testing Zooma annotate service', (t: Test) => {
         console.error(err);
         console.log('annotate error');
         // API call failed...
+        st.notOk(err, 'There is an error');
```

Comment:
@Istar-Eldritch I wanted to check that an error is thrown, but this is the only way I could get it to work. Is there a better way?